fairlearn/postprocessing/__init__.py

```diff
@@ -1,16 +1,22 @@
-# Copyright (c) Microsoft Corporation and Fairlearn contributors.
-# Licensed under the MIT License.
-
-"""This module contains methods which operate on a predictor, rather than an estimator.
-
-The predictor's output is adjusted to fulfill specified parity constraints. The postprocessors
-learn how to adjust the predictor's output from the training data.
-"""
-
-from ._threshold_optimizer import ThresholdOptimizer  # noqa: F401
-from ._plotting import plot_threshold_optimizer  # noqa: F401
-
-__all__ = [
-    "ThresholdOptimizer",
-    "plot_threshold_optimizer"
-]
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""This module contains methods which operate on a predictor, rather than an estimator.
+
+The predictor's output is adjusted to fulfill specified parity constraints. The postprocessors
+learn how to adjust the predictor's output from the training data.
+"""
+
+from ._thresholder import Thresholder  # noqa: F401
+from ._threshold_optimizer import ThresholdOptimizer  # noqa: F401
+from ._plotting import plot_threshold_optimizer, plot_histograms_per_group, \
+     plot_positive_predictions, plot_proba_distr  # noqa: F401
+
+__all__ = [
+    "Thresholder",
+    "ThresholdOptimizer",
+    "plot_threshold_optimizer",
+    "plot_histograms_per_group",
+    "plot_positive_predictions",
+    "plot_proba_distr"

```

Comment:
nit: if we're spelling out "predictions" and "histograms" then "distribution" shouldn't be an issue either 🤣 One could make a case for "hist" and "pos_preds", of course. I'm always a fan of fully spelled out names, but then I'm not like everyone else in this regard... @hildeweerts and other please do feel free to voice your opinions 🙂